src/components/FullScreenPage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+interface FullScreenLayoutProps {
+    title: string;
+   
+  }
+
+
+const FullScreenPage: React.FC<FullScreenLayoutProps> =({title})=>{
```

Comment:
Please, remove a title from the props, we aren't gonna use it. 
Also, add a "children" prop to this component, and render them in return.